### PR TITLE
[MIG] helpdesk_mgmt_category_team: Migration to 15.0

### PR DESCRIPTION
**Pull request by nikunjantala**

### CODE REVIEW

**Review by dsolanki-initos — CHANGES_REQUESTED** [review]
Code review and functional test.
Creating ticket from portal along with attach file at that time an error occurs.

**Comment by jans23** [comment]
> Creating ticket from portal along with attach file at that time an error occurs.

I don't understand this. What error exactly did occur?

**Comment by dsolanki-initos** [comment]
> > Creating ticket from portal along with attach file at that time an error occurs.
> 
> I don't understand this. What error exactly did occur?

Okay let me share video for your reference

https://user-images.githubusercontent.com/50451593/172808621-9346d2a9-bf83-4e8b-8790-fdec4e08f6c9.mp4

**Comment by jans23** [comment]
@nikunjantala Please fix the issue.

**Comment by nileshsheliya** [comment]
@jans23 Issue fixed.

File: helpdesk_mgmt_category_team/controller/main.py [hunk 1]
```diff
@@ -44,7 +44,7 @@ def submit_ticket(self, **kw):
                     request.env['ir.attachment'].sudo().create({
                         'name': c_file.filename,
                         'datas': base64.b64encode(data),
-                        'datas_fname': c_file.filename,
+                        # 'datas_fname': c_file.filename,
```
Comment (jans23) [inline helpdesk_mgmt_category_team/controller/main.py]: In general, code changes should be done without commenting previous parts. Any specific reason for this? if not, simply remove this line.

**Comment by jans23** [comment]
@dsolanki-initos Can you do another test?

**Review by dsolanki-initos — COMMENTED** [review]
Functional test
LGTM except checks failed

**Comment by jans23** [comment]
@dsolanki-initos Check again and resolve your change request if possible.

**Comment by jans23** [comment]
@nikunjantala Please create a PR to OCA.

File: helpdesk_mgmt_category_team/controller/main.py (hunk 1 above)
Comment (nileshsheliya) [reply helpdesk_mgmt_category_team/controller/main.py]: This field is removed in odoo 15.0

**Comment by nileshsheliya** [comment]
> @nikunjantala Please create a PR to OCA.

Need to create PR with which OCA repository?

**Comment by jans23** [comment]
To here: https://github.com/OCA/helpdesk/tree/15.0/

**Comment by nileshsheliya** [comment]
> To here: https://github.com/OCA/helpdesk/tree/15.0/

First I think we need to fork the https://github.com/OCA/helpdesk/tree/15.0/ repo and then we can create PR with OCA repo.

**Comment by jans23** [comment]
Right. Here it is: https://github.com/Nitrokey/odoo-helpdesk/

But fix the checks first and incorporate above feedback before sending to OCA.

File: helpdesk_mgmt_category_team/controller/main.py (hunk 1 above)
Comment (jans23) [reply helpdesk_mgmt_category_team/controller/main.py]: Fine. But then remove it here as well please.

**Comment by nileshsheliya** [comment]
> Right. Here it is: https://github.com/Nitrokey/odoo-helpdesk/
> 
> But fix the checks first and incorporate above feedback before sending to OCA.

As per oca standard, I have fixed all code using oca standard tool pre-commit. So while pushing module on oca, we will not get those errors.

File: helpdesk_mgmt_category_team/controller/main.py (hunk 1 above)
Comment (nileshsheliya) [reply helpdesk_mgmt_category_team/controller/main.py]: It is already removed. This code here showing is already outdated.

**Comment by nileshsheliya** [comment]
I have created PR with OCA repository : https://github.com/OCA/helpdesk/pull/340

**Pull request closed without merging**